### ISSUE TITLE
Error (SQLSTATE[HY000]) while saving 

### ISSUE THREAD

**Issue by magostinelli** [LinkStackOrg/LinkStack#878]
### LinkStack version

4.8.4

### Description

After editing a page, example adding a space between links, i receive the following error:
SQLSTATE[HY000]: General error: 1 table links has no column named type_params 

Query sql involved:
insert into "links" ("title", "button_id", "type_params", "user_id", "id", "updated_at", "created_at") values (5, 43, {"type":"spacer","custom_html":true,"ignore_container":false,"incl

### Details about your system

Debian 11, yunohost 11.3.0.2, php 8.1

### How to reproduce

Insert or edit links, and save

### Additional Context

![immagine](https://github.com/user-attachments/assets/5e3e2acc-b5e8-4952-950c-b95d9ce8722d)

**Comment by tvx-matt** [comment]
@magostinelli 
It would seem a required table is missing from your database. 

Please try executing `your.domain/update?finishing`
This has previously fixed missing tables in the database. 
If this does not work, feel free to ping me and I will take a further look.

**Comment by magostinelli** [comment]
Thanks you for your fast reply, I have already tried with update finishing,  but nothing changed.

**Comment by JulianPrieber** [comment]
You have to run the entire update again. The preparing step has to finish properly.

**Comment by magostinelli** [comment]
Yes, but I Don't know how to restart the database upgrade manually, because the of installati on is managed by yunohost. If I revert back an old backup my users will have the old data in their pages.

**Comment by magostinelli** [comment]
Any news? I tried with /update or

**Comment by JulianPrieber** [comment]
https://github.com/YunoHost-Apps/linkstack_ynh/issues/71#issuecomment-2568058332
